Services/ActiveRecord/Exception/class.arException.php

```diff
@@ -1,5 +1,5 @@
 <?php
-require_once('./Services/Exceptions/classes/class.ilException.php');
+require_once(dirname(__FILE__).'/../../../Services/Exceptions/classes/class.ilException.php');
```

Comment:
Please replace `dirname(__FILE__)` with `__DIR__`. But as written in the other comment, maybe you can just remove the line.